course_settings.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+ECHO 'Hello there.';
+/*
+require_once(__DIR__.'/../../../config.php');
```

Comment:
Perhaps it would be better to finish this page to a working (non explosion) state and refine it later. I generally try to avoid leaving big blocks of uncommented code, even in dev branches